tdrs-backend/tdpservice/search_indexes/parsers/test/test_data_parsing.py

```diff
@@ -0,0 +1,254 @@
+"""Test preparser functions and tanf_parser."""
+import pytest
+from functools import reduce
+from pathlib import Path
+
+from tdpservice.search_indexes.parsers import tanf_parser, preparser, util
+from tdpservice.search_indexes.models import T1
+
+import logging
+logger = logging.getLogger(__name__)
+
+# TODO: ORM mock for true data file factories
+# https://stackoverflow.com/questions/1533861/testing-django-models-with-filefield
+
+@pytest.fixture
+def test_file():
+    """Open file pointer to test file."""
+    test_filepath = str(Path(__file__).parent.joinpath('data'))
+    test_filename = test_filepath + "/small_correct_file"
+    yield open(test_filename, 'rb')
+
+@pytest.fixture
+def test_big_file():
+    """Open file pointer to test file."""
+    test_filepath = str(Path(__file__).parent.joinpath('data'))
```

Comment:
looks like we are already testing big files?

Reply:
Yes although we could go bigger to see what the deployed app can handle.